aspnetcore/mvc/controllers/routing.md

```diff
@@ -70,667 +65,788 @@ public class HomeController : Controller
 
-* the first path segment maps to the controller name
+* The first path segment, `{controller=Home}`, maps to the controller name.
+* The second segment, `{action=Index}`, maps to the [action](#action) name.
+* The third segment, `{id?}` is used for an optional `id`. The `?` in `{id?}` makes it optional. `id` is used to map to a model entity.
 
-* the second maps to the action name.
+Using this `default` route, the URL path:
 
-* the third segment is used for an optional `id` used to map to a model entity
+* `/Products/List` maps to the `ProductsController.List` action.
+* `/Blog/Article/17` maps to `BlogController.Article` and typically model binds the `id` parameter to 17.
 
-Using this `default` route, the URL path `/Products/List` maps to the `ProductsController.List` action, and `/Blog/Article/17` maps to `BlogController.Article`. This mapping is based on the controller and action names **only** and isn't based on namespaces, source file locations, or method parameters.
+This mapping:
 
-> [!TIP]
-> Using conventional routing with the default route allows you to build the application quickly without having to come up with a new URL pattern for each action you define. For an application with CRUD style actions, having consistency for the URLs across your controllers can help simplify your code and make your UI more predictable.
+* Is based on the controller and [action](#action) names **only**.
+* Isn't based on namespaces, source file locations, or method parameters.
+* Is different than typical [attribute routing](#ar) which doesn't include the action name in the route.
+
+Using conventional routing with the default route allows creating the app without having to come up with a new URL pattern for each action. For an app with [CRUD](https://wikipedia.org/wiki/Create,_read,_update_and_delete) style actions, having consistency for the URLs across controllers:
+
+* Helps simplify the code.
+* Makes the UI more predictable.
 
 Most apps should choose a basic and descriptive routing scheme so that URLs are readable and meaningful. The default conventional route `{controller=Home}/{action=Index}/{id?}`:
+
 * Supports a basic and descriptive routing scheme.
 * Is a useful starting point for UI-based apps.
+* Is the only route template needed for many web UI apps. For larger web UI apps, another route using [Areas](#areas) if frequently all that's needed.
+
+<xref:Microsoft.AspNetCore.Builder.ControllerEndpointRouteBuilderExtensions.MapControllerRoute*> and <xref:Microsoft.AspNetCore.Builder.MvcAreaRouteBuilderExtensions.MapAreaRoute*> :
+
+* Automatically assign an **order** value to their endpoints based on the order they are invoked.
+* Don't expose the [RouteEndpoint.Order](xref:Microsoft.AspNetCore.Routing.RouteEndpoint.Order*). `RouteEndpoint.Order` is only exposed for [attribute routing](#ar). <!-- review required for preceding two bullets. Are they OK? -->
```

Comment:
I'm not sure this is the right way to explain this point. Maybe rather than describing attribute routing + order here, leave it for the attribute routing section?

I think the first bullet point is good.